content/docs/start/data-pipelines.md

```diff
@@ -260,22 +261,23 @@ Stage 'prepare' didn't change, skipping
 Stage 'featurize' didn't change, skipping
 ```
 
-Same as before, no need to run `prepare`, `featurize`, etc ... but, it doesn't
-run even `train` again this time either! It cached the previous run with the
-same set of inputs (parameters + data) and reused it.
+This is the same result as before (no need to rerun `prepare`, `featurize`,
+etc.) but it also doesn't run `train` again this time either! DVC maintains a
+<abbr>run-cache</abbr> where it saved the previous run with the same set of
+inputs (parameters + data) for later reuse.
```

Comment:
How about
```suggestion
etc.) but it also doesn't run `train` again this time either! The previous run
with the same set of inputs (parameters + data) was saved in DVC's
<abbr>run-cache</abbr>, and reused here.
```

Reply:
I feel the point here is to explain/sell DVC in general so it makes more sense to keep DVC as the subject of the sentence.

Reply:
Use Cases are more for selling. Get Started is mainly for learning. I expect people who are "already sold" read it through.

Reply:
changed